trio/_core/tests/test_ki.py

```diff
@@ -202,6 +202,18 @@ def protected_manager():
 def test_ki_enabled_out_of_context():
     assert not _core.currently_ki_protected()
 
+def test_ki_disabled_in_del():
+    def nestedfunction():
+            return _core.currently_ki_protected()
```

Comment:
Make sure you run `yapf -rpi setup.py trio` and add the resulting changes to your PR; this will update your code to reflect our whitespace and formatting conventions. (For example, here you have an 8-space indent where our convention is 4.)